src/containers/Payment/index.js

```diff
@@ -35,18 +37,32 @@ const starCount = 5
 class Payment extends Component {
 
     componentDidMount() {
-        this.props.getCheckout()
+        this.props.getCheckout(this.props.match.params.multihash)
+        const redirectUrl = queryString.parse(this.props.location.search).redirect
```

Comment:
We don't seem to be using this, can we not do this parsing here then?

Reply:
Yes, good catch